README.md

```diff
@@ -31,7 +31,7 @@ You can find those interperters here:
 * <del>[ActiveTcl](http://www.activestate.com/activetcl/downloads) 8.6.6</del> (currently disabled)
 * [LuaBinaries](http://luabinaries.sourceforge.net/download.html) 5.4
 * [Python](https://www.python.org/downloads/) 2.7
-* [Python 3](https://www.python.org/downloads/) 3.11
+* [Python 3](https://www.python.org/downloads/) 3.8 or later
```

Comment:
Do we need to explain Python stable ABI in README.md?

Reply:
yes, at least mention that we are using the stable ABI now.

Reply:
Updated. Please let me know if you have better explanation.